hal/renesas-rz.c

```diff
@@ -113,6 +127,50 @@ void ext_flash_unlock(void)
 
 void hal_init(void)
 {
+#if defined(WOLFBOOT_RENESAS_RSIP) && !defined(WOLFBOOT_RENESAS_APP)
+	fsp_err_t err;
+	int ret;
+	rsa_public_t rsip_pub_key;
+	const size_t key_size = sizeof(rsip_pub_key);
+	
+    err = wolfCrypt_Init();
+    if (err != 0) {
+          printf("ERROR: wolfCrypt_Init %d\n", err);
```

Comment:
Check indent. 4 spaces please. Perhaps there are tabs in this file?

Reply:
Thanks. Replaced tabs to 4 spaces.